docs/user-guide/framework-spec.md

```diff
@@ -128,11 +142,17 @@ There can be as many Custom modules as needed.
 
 TBD
 
-## Package Spec
+## Package Definition Pack
```

Comment:
Feels like we can avoid introducing this term. Use `Package` (for PDP) and `Package Manifest`.

Reply:
I think PDP has its own definition, meaning a pack of files exposing package metadata (the manifest is the index file, among the other three). I've pushed an update for this part.

Reply:
I didn't mean we should use it to describe the four files you have labelled PDP. Obviously it cannot mean two things. Can we describe the world effectively without introducing a new label?

Reply:
It's incremental for us but for somebody new it's increasing the total complexity of learning and understanding CDF.

Reply:
You have a point. The way I see it is that they're all integral parts of a package definition therefore grouping them together makes sense, which leads to another naming challenge :) Let me try eliminating the pack definition.

Reply:
idea: maybe referring to those four things _meta data_ when needed is sufficient. I was just trying to think how we might visualise in a diagram.

Reply:
Yep it's between `definitions` and `metadata` both of which I've been using subconsciously anyway. We just need to decide on one.